neural/network.py

```diff
@@ -0,0 +1,42 @@
+from pybrain.tools.shortcuts import buildNetwork
+from pybrain.tools.customxml import NetworkWriter, NetworkReader
+from pybrain.supervised.trainers import BackpropTrainer
+from pybrain.datasets import SupervisedDataSet
+import imgprep
+from functools import reduce
+
+
+class Network():
+    net = None
+
+    def read_from_file(self, file_name):
+        self.net = NetworkReader.readFromFile(file_name)
+
+    def write_to_file(self, file_name):
+        NetworkWriter.writeToFile(net, file_name)
+
+    def init(self, hlayers=2):
```

Comment:
Not really. It builds inner NN, which can be loaded from file as well. There may be more functions of such kind, so perhaps it will be renamed one time.